backend/src/connectors/fileScanning/clamAv.ts

```diff
@@ -2,14 +2,15 @@ import NodeClam from 'clamscan'
 import { Readable } from 'stream'
 
 import { getObjectStream } from '../../clients/s3.js'
-import { FileInterfaceDoc, ScanState } from '../../models/File.js'
+import { FileInterfaceDoc } from '../../models/File.js'
 import log from '../../services/log.js'
 import config from '../../utils/config.js'
 import { ConfigurationError } from '../../utils/error.js'
-import { BaseFileScanningConnector, FileScanResult } from './Base.js'
+import { BaseFileScanningConnector, FileScanResult, ScanState } from './Base.js'
 
 let av: NodeClam
 export const clamAvToolName = 'Clam AV'
+const maxInitRetries = 5
```

Comment:
I'd like the max retry count `maxInitRetries = 5` and delay between pings `10000` to be configurable like `retryDelayInMinutes` e.g. `connectors.fileScanners.maxInitRetries` and `connectors.fileScanners.initRetryDelay`.